rackcity/migrations/0002_itinstance_itmodel_rack_user.py

```diff
@@ -31,7 +33,8 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Rack',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.AutoField(auto_created=True,
```

Comment:
is this the rid? / will it be sent with the other parameters?

Reply:
@clairecfu `id` is an auto-incrementing field that Django adds to every row in a table. we are using that as the primary key, and it will be serialized in all GET requests as `rid` to be consistent with front end naming. didn't want to make a redundant `rid` field since it would serve exactly the same purpose